### ISSUE TITLE
Add more example for binding

### ISSUE THREAD

**Issue by noklam** [altair-viz/altair_notebooks#22]
There are currently no obvious examples about Binding Selection, especially not in "Example Gallery" page.

https://altair-viz.github.io/gallery/index.html

There are some good example already there, it would be great to move them to Example Gallery?
https://altair-viz.github.io/user_guide/interactions.html

**Comment by jakevdp** [comment]
Good point... my feeling is that bindings in Altair 2 are still pretty immature (controlling their appearance/title is not easy, and it's not possible to set a starting value). There's some significant improvement in Vega-Lite 3, so once that's out it will be worth fleshing out the examples a bit more.
